### PR TITLE
Support continuous update distribution

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,7 +23,6 @@
 hs_err_pid*
 
 # Packages inside files folder
-files/*
 !files/.gitkeep
```
Comment (VimukthiPerera) [inline .gitignore]: Shall we remove this and the .gitkeep file as well?

Comment (Savidude) [reply .gitignore]: Done https://github.com/wso2/ansible-is/pull/22/commits/0004d29a774d88a385474cf46f5d6c8b525849de

File: roles/is-analytics-dashboard/tasks/main.yml [hunk 2]
```diff
@@ -15,47 +15,39 @@
...
-        dest: "{{ distribution_path }}"
+        src: "{{ product_package_location }}/packs/{{ product_name }}-{{ product_version }}.zip"
+        dest: "{{ product_dir }}"
         mode: u=rw,g=r,o=r
 
     - name: Copy JDBC Driver
       copy:
-       src: "files/{{ jdbc_driver }}"
-       dest: "{{ install_path }}/lib"
+       src: "files/lib/{{ jdbc_driver }}"
```
Comment (ctienshi) [inline roles/is-analytics-dashboard/tasks/main.yml]: Since we are running using the H2 database by default, shall we comment these?

Comment (Savidude) [reply roles/is-analytics-dashboard/tasks/main.yml]: Fixed https://github.com/wso2/ansible-is/pull/22/commits/c944191a60781df1922abf9b07619ddb62bf61e7

**Pull request merged**